docs/src/tutorial/cython_tutorial.rst

```diff
@@ -127,91 +130,142 @@ Primes
 Here's a small example showing some of what can be done. It's a routine for
 finding prime numbers. You tell it how many primes you want, and it returns
 them as a Python list.
+ 
+.. tabs::
+    .. group-tab:: Cython
 
-:file:`primes.pyx`:
+        .. literalinclude:: ../../examples/tutorial/cython_tutorial/primes.pyx
+            :linenos:
+            :caption: primes.pyx
 
-.. literalinclude:: ../../examples/tutorial/cython_tutorial/primes.pyx
-    :linenos:
+    .. group-tab:: Pure Python
+
+        .. literalinclude:: ../../examples/tutorial/cython_tutorial/primes.py
+            :linenos:
```

Comment:
Couldn't you use linenos to not show the `import cython` that would make tests pass?

Reply:
I think we decided to modify the test-runner to auto-add `import cython`. Although that isn't included in this PR (so currently the tests will probably fail)

Reply:
> Couldn't you use linenos to not show the `import cython` that would make tests pass?
> 
> Edit: or `linenos-start`

Yes, we can, and I have already said this in the issue. The fact is that then all the other lines will be displaced, and it will be very inconvenient to edit them.
Look [here](https://github.com/cython/cython/issues/4187#issuecomment-859973066)